src/core.js

```diff
@@ -1680,11 +1680,17 @@ define([
                 isCopyable: !multiselect && mug.options.isCopyable
             }));
         $baseToolbar.find('.fd-button-remove').click(function () {
-            var mugs = _this.getCurrentlySelectedMug(true);
-            if (mugs.length > 1 || (mugs.length && form.getChildren(mugs[0]).length)) {
+            var mugs = _this.getCurrentlySelectedMug(true),
+                errorMessage;
+            if (mugs.length > 1) {
+                errorMessage = "You are about to delete multiple questions. This cannot be undone";
+            } else if (mugs.length && form.getChildren(mugs[0]).length) {
+                errorMessage = "You are about to delete a question with child nodes. This cannot be undone";
```

Comment:
@millerdev  It was actually in a field report out from Erin, not field/dev. Here's the relevant text

> @Matt/Jon-- interesting! Now definitely makes sense why we are putting the pop-up box for some deletions and and not others. I'm wondering if there's a way we could communicate that (it was kind of confusing to me, and I would presume it might be for a new user, too). Maybe even something as simple as making the delete pop-up box text more specific-- e.g. instead of "you're about to delete a question. Are you sure" something along the lines of "you're about to delete multiple questions and/or questions that have child nodes" (make wording nicer, of course)